infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -110,25 +110,23 @@ module "psoxy-google-workspace-connector" {
   deployment_bundle_object_name         = module.psoxy-gcp.deployment_bundle_object_name
   path_to_config                        = "${local.base_config_path}${each.value.source_kind}.yaml"
   path_to_repo_root                     = var.psoxy_base_dir
-  salt_secret_id                        = module.psoxy-gcp.salt_secret_id
-  salt_secret_version_number            = module.psoxy-gcp.salt_secret_version_number
   example_api_calls                     = each.value.example_api_calls
   example_api_calls_user_to_impersonate = each.value.example_api_calls_user_to_impersonate
   todo_step                             = module.google-workspace-connection[each.key].next_todo_step
 
-  environment_variables =  merge(try(each.value.environment_variables, {}),
+  environment_variables = merge(try(each.value.environment_variables, {}),
     {
       IS_DEVELOPMENT_MODE = contains(var.non_production_connectors, each.key)
     }
   )
 
-  secret_bindings = {
+  secret_bindings = merge({
     # as SERVICE_ACCOUNT_KEY rotated by Terraform, reasonable to bind as env variable
     SERVICE_ACCOUNT_KEY = {
       secret_id      = module.google-workspace-key-secrets[each.key].secret_ids["PSOXY_${replace(upper(each.key), "-", "_")}_SERVICE_ACCOUNT_KEY"]
       version_number = module.google-workspace-key-secrets[each.key].secret_version_numbers["PSOXY_${replace(upper(each.key), "-", "_")}_SERVICE_ACCOUNT_KEY"]
     }
-  }
+  }, module.psoxy-gcp.secrets)
```

Comment:
Now any secrets from `gcp` module are added here instead of being provided as parameter separately